std/experimental/allocator/building_blocks/region.d

```diff
@@ -1114,6 +1113,24 @@ nothrow @nogc:
         return false;
     }
 
+    /**
+    Deallocates all memory allocated by this region, which can be subsequently
+    reused for new allocations.
+    */
+    bool deallocateAll() @trusted pure nothrow @nogc
+    {
+        import core.atomic : atomicStore;
+        static if (growDownwards)
+        {
+            atomicStore(_current, cast(shared(void*)) roundedEnd());
```

Comment:
Do we need the default `MemoryOrder.seq` here rather than `MemoryOrder.rel`?

Reply:
Honestly, I don't know if it will work the same with `MemoryOrder.rel`.
I'd rather not add such an optimization to a function which will be rarely used, at the risk of creating a hard to detect bug.
What are your thoughts on this?
Thanks!